public/components/event_analytics/utils/utils.tsx

```diff
@@ -301,6 +310,61 @@ export const getDefaultVisConfig = (statsToken: statsChunk) => {
   };
 };
 
+export const fetchOtelMetric = async ({
+  visualizationName,
+  startTime,
+  endTime,
+  setIsError,
+  setIsLoading,
+}: {
+  visualizationName: string;
+  startTime: string;
+  endTime: string;
+  setIsLoading: React.Dispatch<React.SetStateAction<boolean>>;
+  setIsError: React.Dispatch<React.SetStateAction<VizContainerError>>;
+}) => {
+  console.log('visualizationName: ', visualizationName);
+  const osdHttp = getOSDHttp();
+  const indexAndDocumentName = extractIndexAndDocumentName(visualizationName);
+  const index = indexAndDocumentName[0];
+  const documentName = indexAndDocumentName[1];
+
+  const fetchSampleDocument = await fetchSampleOTDocument(index, osdHttp, documentName)();
```

Comment:
`fetchSampleDocument` is a verb, it should be a noun like `document`? and what is `OT` in `fetchSampleOTDocument`?

Reply:
OT is for OpenTelemetry